### ISSUE TITLE
Task is restored when I'm the one who comments on it

### ISSUE THREAD

**Issue by cheshire137** [cheshire137/gh-notifications-snoozer#118]
I archive an issue in Snoozer. I then comment on the issue on GitHub. Back in Snoozer, the app auto-refreshes my task list and the issue is no longer archived. I think what's happening is it sees there's a new comment, so it restores the task, even though I'm the one who did the commenting. Not sure if we can get around this. Thoughts @probablycorey?

**Comment by probablycorey** [comment]
Yeah, this is super annoying. The only way I can think of to get around this is by downloading all the comments that are associated with the issue/pr. I'm not sure if this is doable because it would result in N+1 api calls. But if we switched to the GraphQL endpoint, this might be easier to do.
